### ISSUE TITLE
Wind speed is displayed in m/s instead of km/h in weather-forecast card

### ISSUE THREAD

**Issue by Herbs851** [home-assistant/frontend#11080]
### The problem

When using wind speed as secondary_info_attibute in weather-forecast card, speed  is diplayed in m/s instead of km/h.

HA-core 2021.11.5 : 

![weather-1](https://user-images.githubusercontent.com/51154851/147655580-03eaa661-5e06-450f-ab72-5f905bda3fc8.PNG)

HA-core 2021.12.6 : 

![weather-2](https://user-images.githubusercontent.com/51154851/147655596-f3fced9e-d022-4815-ae19-7127d27dd353.PNG)

### What version of Home Assistant Core has the issue?

2021.12.0

### What was the last working version of Home Assistant Core?

2021.11.5

### What type of installation are you running?

Home Assistant Core

### Example YAML snippet

```yaml
### the card's code

type: weather-forecast
entity: weather.my_city
name: my_city
secondary_info_attribute: wind_speed
```

### Additional information

The "wind_speed" attribute of the weather entity has no unit of measurement, HA weather-forecast card seems to use the default speed unit of the metric system (m/s).

I see important changes in the file

```
/lib/python3.9/site-packages/homeassistant/util/unit_system.py
```

since 2021.12.0 version.

That's why I can't tell if the problem comes from the core or the frontend.

**Comment by spacegaier** [comment]
@Herbs851 Can you please copy the weather entity state into the ticket (from the HA dev tools) so we have a basis to reproduce the issue.

**Comment by Herbs851** [comment]
Hi @spacegaier 

The first one with Met.no integration : 

```
temperature: 4.5
humidity: 93
pressure: 1035.4
wind_bearing: 12
wind_speed: 5.8
forecast:
  - condition: cloudy
    precipitation: 0.5
    temperature: 6.5
    templow: 0.9
    datetime: '2022-01-19T11:00:00+00:00'
    wind_bearing: 235.8
    wind_speed: 16.2
  - condition: rainy
    precipitation: 0.4
    temperature: 5.3
    templow: -0.1
    datetime: '2022-01-20T11:00:00+00:00'
    wind_bearing: 354
    wind_speed: 17.3
  - condition: partlycloudy
    precipitation: 0
    temperature: 4.9
    templow: -0.7
    datetime: '2022-01-21T11:00:00+00:00'
    wind_bearing: 12.4
    wind_speed: 10.1
  - condition: partlycloudy
    precipitation: 0
    temperature: 5.6
    templow: -1.4
    datetime: '2022-01-22T11:00:00+00:00'
    wind_bearing: 14.8
    wind_speed: 6.8
  - condition: cloudy
    precipitation: 0
    temperature: 3.3
    templow: -2.1
    datetime: '2022-01-23T11:00:00+00:00'
    wind_bearing: 86.9
    wind_speed: 5.4
attribution: >-
  Weather forecast from met.no, delivered by the Norwegian Meteorological
  Institute.
friendly_name: Maison
```

The second one with Météo-France integration : 

```
temperature: 3.8
humidity: 90
pressure: 1034.6
wind_bearing: 45
wind_speed: 11
forecast:
  - datetime: '2022-01-18T01:00:00+01:00'
    condition: cloudy
    temperature: 6.5
    templow: -1.2
    precipitation: 0
  - datetime: '2022-01-19T01:00:00+01:00'
    condition: cloudy
    temperature: 6.9
    templow: 1.7
    precipitation: 2.1
  - datetime: '2022-01-20T01:00:00+01:00'
    condition: partlycloudy
    temperature: 6.3
    templow: 0.2
    precipitation: 0
  - datetime: '2022-01-21T01:00:00+01:00'
    condition: partlycloudy
    temperature: 5
    templow: -1.7
    precipitation: 0
  - datetime: '2022-01-22T01:00:00+01:00'
    condition: partlycloudy
    temperature: 5.9
    templow: -0.6
    precipitation: 0
  - datetime: '2022-01-23T01:00:00+01:00'
    condition: cloudy
    temperature: 5.2
    templow: 0.7
    precipitation: 0
  - datetime: '2022-01-24T01:00:00+01:00'
    condition: cloudy
    temperature: 5.9
    templow: 1.1
    precipitation: 0
  - datetime: '2022-01-25T01:00:00+01:00'
    condition: cloudy
    temperature: 6.4
    templow: 1.1
    precipitation: 0
  - datetime: '2022-01-26T01:00:00+01:00'
    condition: fog
    temperature: 6
    templow: 1.1
    precipitation: 0
  - datetime: '2022-01-27T01:00:00+01:00'
    condition: fog
    temperature: 5.7
    templow: 0.9
    precipitation: 0
  - datetime: '2022-01-28T01:00:00+01:00'
    condition: partlycloudy
    temperature: 6.1
    templow: 0.5
    precipitation: 0
  - datetime: '2022-01-29T01:00:00+01:00'
    condition: partlycloudy
    temperature: 6.3
    templow: 1.2
    precipitation: 0
  - datetime: '2022-01-30T01:00:00+01:00'
    condition: partlycloudy
    temperature: 6.8
    templow: -0.1
    precipitation: 0
  - datetime: '2022-01-31T01:00:00+01:00'
    condition: partlycloudy
    temperature: 7
    templow: 1.6
    precipitation: 0.1
  - datetime: '2022-02-01T01:00:00+01:00'
    condition: partlycloudy
    temperature: 7.7
    templow: 2.1
    precipitation: 0.2
attribution: Data provided by Météo-France
friendly_name: Saint-Philbert-sur-Boissey
```

let me know if this is what you wanted.

**Comment by spacegaier** [comment]
I think we actually have a general core issue here. 

This PR https://github.com/home-assistant/core/pull/58437 added the (wind) speed unit to the unit systems and apparently selected "m/s" for the metric one. Therefore the frontend uses that unit. 

However, the core dev docs (https://developers.home-assistant.io/docs/core/entity/weather/) state that entities should use "km/h", which is for example what Meteo-France reports here: https://github.com/home-assistant/core/blob/6b85e23408948626bc89a2f2298327769282f736/homeassistant/components/meteo_france/weather.py#L112

So basically all backend integrations that use speeds would need to be updated to return now "m/s"?

@rianadon  @emontnemery Pinging you since you were part of the core PR that set "m/s" as the default.

**Comment by Herbs851** [comment]
@spacegaier 

If I may give my opinion on the choice between m/s and km/h.

From what I can read on different weather websites, and even if m/s is the unit of speed in the metric system, in Germany, Spain, Italy and France the wind speed is well expressed in Km/h.

Perhaps this can be compared to the imperial system which uses miles per hour instead of yards per hour to measure wind speed.

That's why I would like to use Km/h in home-assistant ;)

**Comment by liangleslie** [comment]
> I think we actually have a general core issue here.
> 
> This PR [home-assistant/core#58437](https://github.com/home-assistant/core/pull/58437) added the (wind) speed unit to the unit systems and apparently selected "m/s" for the metric one. Therefore the frontend uses that unit.
> 
> However, the core dev docs (https://developers.home-assistant.io/docs/core/entity/weather/) state that entities should use "km/h", which is for example what Meteo-France reports here: https://github.com/home-assistant/core/blob/6b85e23408948626bc89a2f2298327769282f736/homeassistant/components/meteo_france/weather.py#L112
> 
> So basically all backend integrations that use speeds would need to be updated to return now "m/s"?

Yea seems like @spacegaier assessment of the issue is on point. 

A related issue is that this change has resulted in some unit inconsistencies between core and frontend at the moment.
The weather card shows wind speeds in "m/s" while historical charts show "km/h". But both show the same absolute value (25 in the attached screenshots), even though if we applied conversions, km/h value should be 3.6x larger than m/s.

I don't think there is a standard metric unit for wind speeds, I've seen them reported as km/h and m/s. But maybe the best way forward is to have m/s as the default unit for metric, but with an easy way for devs to declare if they want to use km/h instead for specific integrations.

![weather card](https://user-images.githubusercontent.com/57534857/152076630-c69d85cc-c71c-4535-b62d-5fe8a81e90e5.png)
![wind chart](https://user-images.githubusercontent.com/57534857/152076633-f08a8cb0-3182-4c29-8cd3-9914a68ec952.png)

cc @rianadon @emontnemery to weigh in on the issue.

**Comment by rianadon** [comment]
Sorry I haven't been keeping up to date on notifications here.

The commits I made for standardizing weather units have been reverted in favor of (I think?) being able to configure units per entity. Anyways, reverting #10657 would probably use the right units for most integrations.

**Comment by rjblake** [comment]
> There hasn't been any activity on this issue recently. Due to the high number of incoming GitHub notifications, we have to clean some of the old issues, as many of them have already been resolved with the latest updates. Please make sure to update to the latest Home Assistant version and check if that solves the issue. Let us know if that works for you by adding a comment 👍 This issue has now been marked as stale and will be closed if no further activity occurs. Thank you for your contributions.

This issue is still present. For example, looking at Met.no data, they report a windspeed of 4 m/s, while in Home Assistant the windspeed for the same location is being reported as 14.8 m/s (seems there is a conversion, as 4m/s is roughly 14.4 km/h)

**Comment by emontnemery** [comment]
I think this bug is now fixed by https://github.com/home-assistant/frontend/pull/12946, which will land in Home Assistant Core 2022.7
